server/log_operation_manager.go

```diff
@@ -43,6 +43,8 @@ var (
 	knownLogs    monitoring.Gauge
 	resignations monitoring.Counter
 	isMaster     monitoring.Gauge
+	signingRuns  monitoring.Counter
```

Comment:
If you were feeling energetic you could add a `failedSigningRuns` counter at the same time -- that might be a useful metric.

Reply:
Done.